Update viva-connections-desktop.md (#3478)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/viva-connections-desktop.md

> - The [Viva Connections desktop PowerShell script](https://www.microsoft.com/download/confirmation.aspx?id=102888) is available now in the [Microsoft download center](https://www.microsoft.com/download/default.aspx).
> - The vanity domain is only supported if the home site has the same domain as the [root site](modern-root-site.md).
> - Viva Connections is not supported on the Linux operating system.

> > - Viva Connections desktop is not supported on mobile or tablet devices. Please use the upgraded app from [Set up and launch Viva Connections](https://learn.microsoft.com/en-us/viva/connections/guide-to-setting-up-viva-connections) to include the mobile experience. 

Reviewer: ```suggestion
> - Viva Connections desktop is not supported on mobile or tablet devices. Use the upgraded app described in [Set up and launch Viva Connections](https://learn.microsoft.com/viva/connections/guide-to-setting-up-viva-connections) to include the mobile experience. 
```
